src/QirTools/pyqir/qirlib/src/emit/mod.rs

```diff
@@ -0,0 +1,276 @@
+use self::constants::Constants;
+use self::intrinsics::Intrinsics;
+use self::runtime_library::RuntimeLibrary;
+use self::types::Types;
+use inkwell::values::BasicValueEnum;
+
+use super::interop::SemanticModel;
+use std::collections::BTreeMap;
+use std::path::Path;
+
+pub mod constants;
+    mod intrinsics;
```

Comment:
Is this standard `rustfmt`? I'd suggest adding a `rustfmt` check to CI passes regardless just to make sure we're using a consistent style.